include/cantera/numerics/Func1.h

```diff
@@ -993,11 +1191,12 @@ class Arrhenius1 : public Func1
         return *this;
     }
 
-    virtual Func1& duplicate() const {
-        Arrhenius1* np = new Arrhenius1(*this);
-        return *((Func1*)np);
+    virtual string type() const {
+        return "arrhenius";
```

Comment:
If we want to stick to the naming convention that we've used elsewhere (YAML), then we should capitalize proper names like Arrhenius, Gaussian, and Fourier.

Reply:
Was basing nomenclature on Matlab, but I'm :+1: with this change.